agent/containers/images/change-perm

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+AUTH="${1}"
+
+function change {
+    echo https://quay.io/api/v1/repository/pbench/$1/permissions/team/core
+    curl \
+        -X PUT \
+        -H "Content-Type: application/json" \
+        -H "Authorization: Bearer $AUTH" \
+        -d '{ "role": "admin" }' \
+        https://quay.io/api/v1/repository/pbench/$1/permissions/team/core
+}
+
+for i in workloads tools tool-meister tool-data-sink base all; do
+    change "pbench-agent-${i}-fedora-33"
+    change "pbench-agent-${i}-fedora-34"
+    change "pbench-agent-${i}-centos-7"
+    change "pbench-agent-${i}-centos-8"
```

Comment:
This is a highly-opinionated (and possibly stale!) list; can't we do a GET to find out what is there, and then iterate over the result?